test/controllers/user.controller.test.js

```diff
@@ -811,21 +811,83 @@ describe('User Controller', () => {
   });
 
   describe('Get all users', () => {
+    const dummyUsers = UserFactory.buildList(18);
+    beforeEach(async () => {
+      await User.insertMany(dummyUsers);
+    });
+
     describe('when users exist in db', () => {
-      context('with a valid token & id', () => {
-        it('returns successful payload', (done) => {
+      context('when no parameters are passed', () => {
+        it('returns first page with ten items', (done) => {
           request()
             .get('/api/v1/user/all')
             .set('authorization', adminToken)
             .end((err, res) => {
               expect(res).to.have.status(200);
               expect(res.body.success).to.be.eql(true);
-              expect(res.body.result.length).to.be.eql(2);
+              expect(res.body.result.length).to.be.eql(10);
               expect(res.body.pagination.currentPage).to.be.eql(1);
               expect(res.body.pagination.limit).to.be.eql(10);
               expect(res.body.pagination.offset).to.be.eql(0);
-              expect(res.body.pagination.total).to.be.eql(2);
-              expect(res.body.pagination.totalPage).to.be.eql(1);
+              expect(res.body.pagination.total).to.be.eql(20);
+              expect(res.body.pagination.totalPage).to.be.eql(2);
+              done();
+            });
+        });
+      });
+
+      context('when parameters page and limit are passed', () => {
+        it('returns second page with five items', (done) => {
+          request()
+            .get('/api/v1/user/all?page=2&limit=5')
+            .set('authorization', adminToken)
+            .end((err, res) => {
+              expect(res).to.have.status(200);
+              expect(res.body.success).to.be.eql(true);
+              expect(res.body.result.length).to.be.eql(5);
+              expect(res.body.pagination.currentPage).to.be.eql('2');
```

Comment:
Can we make this a number so that it can be the same with the other parameters: Please fix for other assertions too.
```suggestion
              expect(res.body.pagination.currentPage).to.be.eql(2);
```